sys/include/net/fib.h

```diff
@@ -63,6 +63,11 @@ typedef struct fib_destination_set_entry_t {
 #define FIB_LIFETIME_NO_EXPIRE (0xFFFFFFFFffffffff)
 
 /**
+ * @brief flag to identify if a route was set by RPL
+ */
+#define FIB_FLAG_RPL_ROUTE (1UL << 0)
```

Comment:
while I agree with your idea, I suggest we postpone the enum definition since we still do not really know how many flags we will need and if they need to be exclusive or if they can be ORed together. What do you think?